src/components/side_panel/chart/building_blocks/general_design/general_design_editor.ts

```diff
@@ -65,8 +66,12 @@ export class GeneralDesignEditor extends Component<Props, SpreadsheetChildEnv> {
   }
 
   get titleStyle(): TitleDesign {
+    const isScorecardOrGauge = ["scorecard", "gauge"].includes(this.props.definition.type);
     return {
       align: "left",
+      fontSize:
+        this.title.fontSize ??
+        (isScorecardOrGauge ? SCORECARD_GAUGE_CHART_FONT_SIZE : DEFAULT_CHART_FONT_SIZE),
```

Comment:
I think it'd be better to add a `defaultTitleFontSize` props rather than this. It's better for the GeneralDesignEditor to be totaly unaware of chart types and their specificities